spec/controllers/test_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe TestController, type: :controller do
+  describe '#create' do
+    let(:valid_params) { { name: 'Test Name', time_to_do: 60, time_start: Time.current } }
+    let(:invalid_params) { { name: '', time_to_do: 60, time_start: Time.current } }
+
+    context 'when input data is valid' do
+      it 'returns a success response with message "ok"' do
+        mock_service = instance_double(TestService::Create, call: nil, errors: [],error?: false)
+        expect(TestService::Create).to receive(:new).and_return(mock_service)
```

Comment:
There is an issue with this expect: it doesn't assert that the service object must be called with correct parameters.

Reply:
yes, i'm trying to fix that, i used with method but it doesn't work, I'll push later